src/app/models/cake.model.ts

```diff
@@ -0,0 +1,34 @@
+export const enum statusCake {
```

Comment:
Since we are using a Single File Responsibility file distribution, this Enumerator should live in a separate file, maybe named `cake-status.enum.ts`. 

The enums should be named following the JavaScript Style Guides, I suggest to change the name to `CakeStatus`.

Reply:
Sure, I will change this